sdk/storage/azure-storage-files-shares/src/share_file_client.cpp

```diff
@@ -272,8 +272,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
         auto newResponse = Download(newOptions, context);
         if (eTag != newResponse.Value.Details.ETag)
         {
-          throw Azure::Core::RequestFailedException(
-              "File was changed during the download process.");
+          throw Azure::Core::RequestFailedException("file was modified in the middle of download");
```

Comment:
```suggestion
          throw Azure::Core::RequestFailedException("File was modified in the middle of download.");
```

Reply:
We need to be consistent about exception messages (uppercase or lowercase for the first character, `.` at the end). What's the format of exception message in Core SDK?

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/2366

Reply:
There might be mistakes in core about this (will check and see), but we should be using complete sentences that are customer-oriented so the developer can understand what went wrong and how they might need to handle those cases.

This is a good example (especially since we usability tested it, even though it may not be an exception):
https://github.com/Azure/azure-sdk-for-cpp/blob/dfd2933a66372bcc27e8563f5ef2329615f3a894/sdk/core/azure-core/inc/azure/core/io/body_stream.hpp#L61-L68